sigs/community/CHARTER.md

```diff
@@ -0,0 +1,28 @@
+# Proposed name: SIG Community
+
+## Objective
+
+A platform for students and new developers looking to get into TensorFlow, as well
```

Comment:
We used Gitter for communications during last year's GSoC and I found it very sufficient already. Why would a platform and a new SIG be necessary for objective like this?

Reply:
Maybe GSoC had a platform, but GCI didn't.